torch/csrc/jit/passes/quantization.cpp

```diff
@@ -403,8 +403,8 @@ Node* insertQuantDeQuantCall(
   if (is_per_channel) {
     scale_val = g->insertConstant(scale);
     zero_point_val = g->insertConstant(zero_point);
-    // TODO: get axis from qparam
-    axis_val = g->insertConstant(0);
+    int axis = tp->elements()[2].toInt();
```

Comment:
This should be squashed with the change where it was introduced before landing. There is no reason to commit half-finished stuff.

Reply:
this is just because the order we do things right now, at the place we introduce this, calculate_qparam doesn't return axis

Reply:
Ok, I reread everything more carefully and it makes sense.